orte/mca/rml/ofi/rml_ofi_component.c

```diff
@@ -550,52 +622,44 @@ orte_rml_ofi_progress(ofi_transport_conduit_t* conduit)
  */
 int cq_progress_handler(int sd, short flags, void *cbdata)
 {
-    ofi_transport_conduit_t* conduit = (ofi_transport_conduit_t*)cbdata;
+    ofi_transport_ofi_prov_t* prov = (ofi_transport_ofi_prov_t*)cbdata;
     int count;
 
     opal_output_verbose(1, orte_rml_base_framework.framework_output,
-                         "%s cq_progress_handler called for OFI conduitid %d",
+                         "%s cq_progress_handler called for OFI Provider id %d",
                          ORTE_NAME_PRINT(ORTE_PROC_MY_NAME),
-                         conduit->conduit_id);
+                         prov->ofi_prov_id);
 
       /* call the progress fn to read the cq and process the message
-        *    for the conduit */
-        count = orte_rml_ofi_progress(conduit);
+        *    for the ofi provider */
+        count = orte_rml_ofi_progress(prov);
         return count;
 }
 
 
-static orte_rml_base_module_t*
-rml_ofi_component_init(int* priority)
+/* 
+ * Returns the number of ofi-providers available
+ */
+int rml_ofi_component_init()
```

Comment:
It feels like this should be a static function.  And the parameter list should be `(void)`.